content/docs/reference/providers/pulumi/aws/s3/bucket.md

```diff
@@ -0,0 +1,1827 @@
+
+```python
+def __init__(__self__, resource_name, opts=None, acceleration_status=None, acl=None, arn=None, bucket=None, bucket_prefix=None, cors_rules=None, force_destroy=None, hosted_zone_id=None, lifecycle_rules=None, loggings=None, object_lock_configuration=None, policy=None, region=None, replication_configuration=None, request_payer=None, server_side_encryption_configuration=None, tags=None, versioning=None, website=None, website_domain=None, website_endpoint=None, __props__=None)
+```
+
+```go
+func NewBucket(ctx *pulumi.Context, name string, args *BucketArgs, opts ...pulumi.ResourceOption) (*Bucket, error)
+
+```
+
+```csharp
+public Bucket(string name, BucketArgs? args = null, CustomResourceOptions? options = null)
+
+```
+
+Creates a Bucket resource with the given unique name, arguments, and options.
+
+{{% lang nodejs %}}
+<ul class="pl-10">
+    <li><strong>name</strong> &ndash; (Required) The unique name of the resulting resource.</li>
+    <li><strong>args</strong> &ndash; (Optional) The arguments to use to populate this resource's properties.</li>
+    <li><strong>opts</strong> &ndash; (Optional) A bag of options that control this resource's behavior.</li>
+</ul>
+{{% /lang %}}
+
+{{% lang go %}}
+<ul class="pl-10">
+    <li><strong>name</strong> &ndash; (Required) The unique name of the resulting resource.</li>
+    <li><strong>args</strong> &ndash; (Optional) The arguments to use to populate this resource's properties.</li>
+    <li><strong>opts</strong> &ndash; (Optional) A bag of options that control this resource's behavior.</li>
+</ul>
+{{% /lang %}}
+
+{{% lang csharp %}}
+<ul class="pl-10">
+    <li><strong>name</strong> &ndash; (Required) The unique name of the resulting resource.</li>
+    <li><strong>args</strong> &ndash; (Optional) The arguments to use to populate this resource's properties.</li>
+    <li><strong>opts</strong> &ndash; (Optional) A bag of options that control this resource's behavior.</li>
+</ul>
+{{% /lang %}}
+
+The following arguments are supported:
+
+<table class="ml-6">
+    <thead>
+        <tr>
+            <th>Argument</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td class="align-top">acceleration<wbr>Status</td>
```

Comment:
Short we sort the required properties to come first? I'd also love to make them pop better visually, either an icon or color the rows differently.